### PR TITLE
CarlKho PCW Intro MD for Scheffler MW@11AM

### PR DESCRIPTION
**Pull request by CarlKho-Minerva**
CarlKho PCW Intro MD for Scheffler MW@11AM

### CODE REVIEW

**Comment by 000alen** [comment]
consider using the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format in the future

**Review by jonathanshamwana — APPROVED** [review]
I think you should cold email Sam Altman. Get him to merge your changes

File: 2024/fall/Scheffler-MW@11am/carl-kho.md
Comment (000alen) [inline 2024/fall/Scheffler-MW@11am/carl-kho.md:1]: lgtm

File: 2024/fall/Scheffler-MW@11am/carl-kho.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Carl Kho
```
Comment (000alen) [inline 2024/fall/Scheffler-MW@11am/carl-kho.md:1]: nitpick: the top-level header is supposed to be a markdown `h1`

**Review by 000alen — APPROVED** [review]

**Comment by CarlKho-Minerva** [comment]
> I think you should cold email Sam Altman. Get him to merge your changes

ill lyk what happens haha

**Pull request merged**